src/main.cpp

```diff
@@ -6794,6 +6793,107 @@ ThresholdState VersionBitsTipState(const Consensus::Params& params, Consensus::D
     return VersionBitsState(chainActive.Tip(), params, pos, versionbitscache);
 }
 
+bool LoadMempool(void)
+{
+    FILE* filestr = fopen((GetDataDir() / "mempool.dat").c_str(), "r");
+    CAutoFile file(filestr, SER_DISK, CLIENT_VERSION);
+    if (file.IsNull()) {
+        return error("Failed to open mempool file from disk");
```

Comment:
I don't think missing this file should be an error. I remember users being confused by errors about other files missing (peers.dat), which are simply generated by the application anyhow.